### PR TITLE
Drop Python<3.5 and adopt src layout

### PR DESCRIPTION
**Pull request by nicoddemus**
- Update classifiers
- Remove setup.cfg as it is no longer necessary
- Remove `CONTRIBUTING.rst` as it is out dated and leads to confusion
  (development instructions are available in the README itself)
- Executed 'pyupgrade --py36-plus'

### CODE REVIEW

File: src/esss_fix_format/cli.py [hunk 1]
```diff
@@ -210,7 +209,7 @@ def _process_file(filename, check, format_code):
             try:
                 subprocess.check_output('clang-format -i "%s"' % filename, shell=True)
             except subprocess.CalledProcessError as e:
-                msg = ': ERROR (%s: %s): ' % (type(e).__name__, e)
+                msg = ': ERROR ({}: {}): '.format(type(e).__name__, e)
```
Comment (williamjamir) [inline src/esss_fix_format/cli.py]:
```suggestion
                msg = f': ERROR ({type(e).__name__}: {e}): '
```

File: src/esss_fix_format/cli.py [hunk 2]
```diff
@@ -233,7 +232,7 @@ def _process_file(filename, check, format_code):
             f.seek(0)
             original_contents = f.read()
         except UnicodeDecodeError as e:
-            msg = ': ERROR (%s: %s)' % (type(e).__name__, e)
+            msg = ': ERROR ({}: {})'.format(type(e).__name__, e)
```
Comment (williamjamir) [inline src/esss_fix_format/cli.py]:
```suggestion
            msg = f': ERROR ({type(e).__name__}: {e})'
```

File: src/esss_fix_format/hook_utils.py [hunk 3]
```diff
@@ -88,10 +86,10 @@ def install_pre_commit_hook(git_dir=None):
 
     git_root = os.path.abspath(git_root)
     project_name = os.path.basename(git_root)
-    print('{0} hooks'.format(project_name))
+    print(f'{project_name} hooks')
 
     if not os.path.exists(os.path.join(git_root, '.git')):
-        raise ValueError('Expected to find: %s' % (os.path.join(git_root, '.git'),))
+        raise ValueError('Expected to find: {}'.format(os.path.join(git_root, '.git')))
```
Comment (williamjamir) [inline src/esss_fix_format/hook_utils.py]:
```suggestion
        raise ValueError('Expected to find: {os.path.join(git_root, '.git')}')
```

**Comment by nicoddemus** [comment]
Skipping the formatting suggestions as I want to leave only the automated changes made by `pyupgrade`. 👍

**Pull request merged**